### ISSUE TITLE
[Hardware] Set up Ubuntu server for collaborative testing

### ISSUE THREAD

**Issue by jamieyello** [MythologyScape/MythScape#3]
This is for CryptoGam3r AKA @MiniScaper to set up the development server. This is lower priority, but once issue #2 is complete we can move on to deployment. 

 - [ ] Install Ubuntu server on the PC. (Use Ubuntu (desktop version) on an 8GB USB flash drive to live boot and wipe the SSD before installing Ubuntu Server)
 - [ ] Set up SSH so that @jamieyello (me) can access it remotely.
 - [ ] Port forward a couple ports. We will work out the specific ports when the time comes, it shouldn't be more than 4.

**Comment by jamieyello** [comment]
Closing as stale
